test/built-ins/RegExp/incomplete_hex_unicode_escape.js

```diff
@@ -0,0 +1,16 @@
+// This code is governed by the BSD license found in the LICENSE file.
```

Comment:
@ljharb @littledan I'm upset we need to request a fix for this. I wish this could pass without being an issue.

@Zirak unfortunetly, this project still requires a 2 lines BSD header, you need to mention an explicit Copyright.

Reply:
Done.